lib/index.ts

```diff
@@ -27,8 +27,29 @@ export async function percySnapshot(page: Page, name: string, options: any = {})
   await page.addScriptTag({
     path: agentJsFilename()
   })
-  await page.evaluate(function(name: string, options: any, clientInfo: string) {
-    const percyAgentClient = new PercyAgent({ clientInfo })
-    percyAgentClient.snapshot(name, options)
+
+  if (! await isAgentRunning()) {
+    return
+  }
+
+  const domSnapshot = await page.evaluate(function(name: string, options: any, clientInfo: string) {
```

Comment:
as discussed, we'll remove the `clientInfo` references here since they don't actually take effect.